string droneID
string sessionID
GPS gps
float64 batteryLevel
int32 status
string messageType